R/zchunk_L221.land_input_1.R

```diff
@@ -1,20 +1,38 @@
 #' module_aglu_L221.land_input_1
 #'
-#' Briefly describe what this chunk does.
+#' This chunk produces L221.LN0_Logit - Logit exponent of the top-level (zero) land nest by region; L221.LN0_Land - Total regional land allocation in top level nest;
+#' L221.LN0_SoilTimeScale - Soil time scale by region; L221.LN1_ValueLogit - Unmanaged land value by region and GLU, and logit exponent of first nest;
+#' L221.LN1_HistUnmgdAllocation - Historical land cover fora unmanaged land (LT_GLU) in the first nest by region;
+#' L221.LN1_UnmgdAllocation - Land cover in the model base periods for unmanaged land (LT_GLU) in the first nest by region; and
+#' L221.LN1_UnmgdCarbon - Carbon content for unmanaged land (LT_GLU) in first nest by region.
```

Comment:
I'd prefer the top-level description be a single sentence or two. Save this for the `@details` please.